ckanext/saml2auth/views/saml2auth.py

```diff
@@ -130,9 +130,11 @@ def process_user(email, saml_id, full_name, saml_attributes):
 
         current_user_dict = copy.deepcopy(user_dict)
 
-        if email != user_dict['email'] or full_name != user_dict['fullname']:
+        # We have removed the fullname check and update since full proofing isn't used.
+        # If we don't remove this, their display names will always be overwritten with the email addresses.
+        if email != user_dict['email']: # and full_name != user_dict['fullname']:
             user_dict['email'] = email
-            user_dict['fullname'] = full_name
+            # user_dict['fullname'] = full_name
```

Comment:
are we leaving this here as legacy code?

Reply:
Yeah. Leaving that there in case they decide to do full proofing (which means their name will always match the full name on login.gov)